vitest.env.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module '*.gif' {
```

Comment:
this is needed for imports of images to be recognized when we import them for browser mode tests